### PR TITLE
Add support for headers to be specified in the documentation

### PR DESCRIPTION
**Pull request by jaischeema**
https://www.pivotaltracker.com/story/show/108834704

screen shot 2015-11-25 at 11 18 58 am

### CODE REVIEW

File: lib/grape_slate/generators/headers.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      def route_header_examples
+        Hash[route_headers.map { |key, info_hash| [key, info_hash[:documentation][:example]] }]
+      end
+
+      def generate
+        array = []
+
+        unless route_headers.empty?
+          array << "### Request Headers"
+          array << "Header | Required / Values | Description"
```
Comment (wezm) [inline lib/grape_slate/generators/headers.rb]: Can we split Required and Values?

**Comment by cdaloisio** [comment]
LGTM :+1:

File: lib/grape_slate/generators/headers.rb (hunk 1 above)
Comment (jaischeema) [inline lib/grape_slate/generators/headers.rb]: Will do.

**Comment by wezm** [comment]
Looks good 👍

**Comment by psiedlak** [comment]
+1

**Pull request merged**